src/main/java/uk/gov/digital/ho/hocs/client/casework/CaseworkClient.java

```diff
@@ -48,4 +49,11 @@ public UUID getPrimaryCorrespondent(UUID caseUUID) {
         ResponseEntity<String> responseEntity = restClient.get(serviceBaseURL, String.format("/case/%s", caseUUID), String.class);
         return UUID.fromString(JsonPath.read(responseEntity.getBody(), "$.primaryCorrespondentUUID"));
     }
+
+    public void updateStageTeam(UUID caseUUID, UUID stageUUID, UUID teamUUID) {
```

Comment:
Should this have `ResponseEntity<void>` return type and return the entity rather than being a void method, re consistency with other methods in file

Reply:
Changed to ResponseEntity<Void> and added missing test.